Source/WebCore/inspector/InspectorFrontendHost.cpp

```diff
@@ -860,4 +861,28 @@ void InspectorFrontendHost::setPath(CanvasRenderingContext2D& context, Path2D& p
     context.setPath(path);
 }
 
+#if ENABLE(OFFSCREEN_CANVAS)
+
+float InspectorFrontendHost::getCurrentX(const OffscreenCanvasRenderingContext2D& context) const
```

Comment:
I'm getting a sense of deja vu, but I still wish these method names were more specific (e.g. `getCanvasCurrentX`). But I know there is a reason we didn't that I've since forgotten.